wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php

```diff
@@ -273,8 +275,14 @@ protected function parse_readme_contents( $contents ) {
 			$this->tags = explode( ',', $headers['tags'] );
 			$this->tags = array_map( 'trim', $this->tags );
 			$this->tags = array_filter( $this->tags );
-			$this->tags = array_diff( $this->tags, $this->ignore_tags );
-			$this->tags = array_slice( $this->tags, 0, 5 );
+			if ( array_intersect( $this->tags, $this->ignore_tags ) ) {
+				$this->tags = array_diff( $this->tags, $this->ignore_tags );
+				$this->warnings['ignored_tags'] = true;
```

Comment:
I noticed this `ignored_tags` is added here. Nice addition. Is it possible to expose the list of ignored tags so that we could show relevant message if those tags are added? Currently `$ignore_tags` is `private`.

Reply:
Changing `$ignore_tags` to public shouldn't be an issue.